activity/activity.info

```diff
@@ -1,7 +1,7 @@
 [Activity]
 name = JAMClock
 activity_version = 2
-bundle_id = org.sugarlabs.JAMClock
+bundle_id = org.laptop.JAMClock
 icon = JAMClock
 show_launcher = yes
-exec = sugar-activity3 JAMClock.JAMClock
+exec = JAMClock.JAMClock
```

Comment:
`exec` must contain either sugar-activity or sugar-activity3; you've removed both, and this causes the traceback you observed;

```log
Traceback (most recent call last):
File "/usr/lib/python2.7/dist-packages/sugar3/activity/activityinstance.py", line 175, in main
splitted_module = activity_class.rsplit('.', 1)
AttributeError: 'NoneType' object has no attribute 'rsplit'   
```